app/controllers/providers_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
   end
 
   def show
-    @has_multiple_providers = Provider.all.size > 1
+    @has_multiple_providers = session.to_hash.dig(:auth_user, :provider_count).to_i > 1
```

Comment:
🤓 http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/#arraydig-and-hashdig

Reply:
`nil.to_i # => 0`. is that the right default if the thing isn't set for some reason?

Reply:
Good question, maybe not... rather always show the breadcrumbs I think as not having them is more harmful than having useless ones?

Reply:
agree that "show if in doubt" is safer

Reply:
Updated

Reply:
This whole `dig` thing actually ended up working incorrectly so I swapped it to `.try`...